python/tvm/script/tir/__init__.pyi

```diff
@@ -465,13 +465,13 @@ class uint64(PrimExpr):
     def __init__(self: uint64, imm: Union[PrimExpr, int]): ...
 
 class float8(PrimExpr):
-    def __init__(self: float8, imm: Union[PrimExpr, int, float]): ...
+    def __init__(self: float8, imm: Union[PrimExpr, int, float, str]): ...
```

Comment:
would be better to explicitly define a list of special literals:
```
SpecialFloatLiteral = Literal["inf", "-inf", "nan"]
```

Reply:
right now tvm only requires python >= 3.6, which means `Literal` may not be available if python <3.8 is used. What about we put something like `SpecialFloatLiteral: str = str["inf", "-inf", "nan"]` until python>=3.8 is implemented in tvm?

Reply:
Sounds good

Reply:
seems like `typing_extension.Literal` can be used for python < 3.8. This is updated.